src/main/resources/assets/twilightforest/lang/en_us.lang

```diff
@@ -83,6 +83,7 @@ advancement.twilight_progress_castle.desc=What could even be in that castle?!?
 twilightforest.trophyPedestal.ineligible=You are unworthy.
 twilightforest.ore_meter.range=Radius: %s, Origin: [%s, %s]
 twilightforest.ore_meter.exposed=Exposed: %s
+twilightforest:scepter_charges=%d charges left
```

Comment:
colons won't work with `#PARSE_ESCAPES`, i'd use `.` instead anyway to match everything else

Reply:
heck, i had illusion and thought all those dot above are all colons. will be fixed right now

Reply:
uh... turns out that it's the `zh_cn.lang` that being outdated and still using colon...